### ISSUE TITLE
Add extra metadata on assets about last modified and author

### ISSUE THREAD

**Issue by yaustar** [playcanvas/editor#363]
What would be useful in keeping track of assets is to show the last modified date/time and who it was if possible. Almost like a 'blame' too.

**Comment by yaustar** [comment]
Related to: https://github.com/playcanvas/editor/issues/330

**Comment by Maksims** [comment]
What is considered last modified? When ticking "preload" or renaming asset would update last modified?
Or this would be more useful regarding file update (I believe this is more of a case).

File Modified - probably would be more useful. It would be a date of creation or subsequent upload.
This field would not be available for: folder.
Also, few assets like material do not have file related to them, although can be sourced from container/fbx, which means it has "file" notion.

**Comment by yaustar** [comment]
Yeah, not entirely sure it it's needed at the asset meta level but the actual file (png) would definitely help
